src/ROUTES.js

```diff
@@ -2,19 +2,18 @@
  * This is a file containing constants for all of the routes we want to be able to go to
  */
 export default {
-    HOME: '/home',
+    HOME: '/',
     SETTINGS: '/settings',
     SETTINGS_PROFILE: '/settings/profile',
     SETTINGS_PREFERENCES: '/settings/preferences',
     SETTINGS_PASSWORD: '/settings/password',
     SETTINGS_PAYMENTS: '/settings/payments',
     NEW_GROUP: '/new/group',
     NEW_CHAT: '/new/chat',
+    REPORT: '/r',
     IOU_REQUEST: '/iou/request',
     IOU_BILL: '/iou/split',
-    REPORT: '/r/:reportID',
     getReportRoute: reportID => `/r/${reportID}`,
-    ROOT: '/',
     SEARCH: '/search',
     SET_PASSWORD: '/setpassword/:validateCode',
```

Comment:
It looks like some of these route definitions are no longer used. Can these be cleaned up?